qtconsole/completion_widget.py

```diff
@@ -85,16 +56,24 @@ def eventFilter(self, obj, event):
                 else:
                     self.cancel_completion()
                     if target:
-                        target.event(event)
+                        try:
+                            target.event(event)
+                        except RuntimeError:
+                            pass
```

Comment:
I was getting a `RuntimError` when moving focus out of the completion with a mouse click on the main qtconsole window.